flink-java/src/main/java/org/apache/flink/api/java/ExecutionEnvironment.java

```diff
@@ -144,17 +144,16 @@ protected ExecutionEnvironment() {
 	}
 
 	protected ExecutionEnvironment(final Configuration configuration) {
-		this(DefaultExecutorServiceLoader.INSTANCE, configuration);
+		this(DefaultExecutorServiceLoader.INSTANCE, configuration, null);
```

Comment:
instead of `null`, shall we directly pass `getClass().getClassLoader()`?

Reply:
That would be ideal, but the problem is that this is not allowed by Java. You cannot call `getClass()` before the `super` constructor is called.

Reply:
Is it possibly we get the same classloader in another way?